src/main/java/com/crowdin/client/sourcefiles/model/ImportOptions.java

```diff
@@ -1,4 +1,8 @@
 package com.crowdin.client.sourcefiles.model;
 
+import lombok.Data;
+
+@Data
 public abstract class ImportOptions {
+    private Boolean firstLineContainsHeader;
```

Comment:
That can't be here. The `firstLineContainsHeader` is relevant only for the `SpreadsheetFileImportOptions` and not for base `ImportOptions` which is also extended by other classes that do not need `firstLineContainsHeader`.